scripts/audit-consumers.ts

```diff
@@ -0,0 +1,201 @@
+  // In short, uppercase the first letter in each sequence of letters and remove hyphens.
+  const expectedCounterLabels = new Set(
+    Object.keys(webFeatures).map((id) => {
+      return id
+        .replace(/[a-z]+/g, (m) => m[0].toUpperCase() + m.substring(1))
+        .replaceAll("-", "");
+    }),
+  );
+
+  const items = [];
+
+  for (let [label, value] of counters.entries()) {
+    if (label == "PageVisits" && value == 0) {
+      continue;
+    }
+    if (label.startsWith("OBSOLETE_")) {
+      continue;
+    }
+    const draft = label.startsWith("DRAFT_");
+    if (draft) {
+      label = label.substring(6);
+    }
+    if (!expectedCounterLabels.has(label)) {
+      if (draft) {
+        items.push(`${label} (draft)`);
+      } else {
+        items.push(`${label} (not draft)`);
+      }
+    }
+  }
+
+  return {
+    heading: "Chromium use counters",
+    items,
+    trailer:
+      "Source: [webdx_feature.mojom](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/public/mojom/use_counter/metrics/webdx_feature.mojom)",
+  };
+}
+
+async function chromeStatusReport() {
+  const items = [];
+
+  for await (const f of chromeStatusFeatures()) {
+    const id = f.web_feature;
+    if (!id || id === "Missing feature") {
+      continue;
+    }
+    if (!Object.hasOwn(webFeatures, id)) {
+      items.push(`${id} used by https://chromestatus.com/feature/${f.id}`);
+    }
+  }
+
+  return {
+    heading: "Chromestatus entries",
+    items,
+    trailer: "Source: [chromestatus.com](https://chromestatus.com/)",
+  };
+}
+
+async function wptReport() {
+  const octokit = new Octokit();
+
+  const params = { owner: "web-platform-tests", repo: "wpt" };
+  const release = await octokit.rest.repos.getLatestRelease(params);
+
+  if (!release) {
+    throw new Error(
+      `No latest release of ${params.owner}/${params.repo} found`,
+    );
+  }
+
+  const label = "WEB_FEATURES_MANIFEST.json.gz";
+  const asset = release.data.assets.find((asset) => asset.label == label);
+
+  if (!asset) {
+    throw new Error(`No ${label} asset found in ${release.data.html_url}`);
+  }
+
+  const response = await fetch(asset.browser_download_url);
+  if (!response.ok) {
+    throw new Error(`Failed to download ${asset.browser_download_url}`);
+  }
+
+  const gunzip = zlib.createGunzip();
+  const chunks = [];
+  gunzip.on("data", (chunk) => chunks.push(chunk));
+  await stream.pipeline(response.body, gunzip);
+  const json = Buffer.concat(chunks).toString("utf-8");
+
+  const manifest = JSON.parse(json);
+  const ids = Object.keys(manifest.data);
+
+  const items = [];
+  for (let id of ids) {
+    const draft = id.startsWith("draft/");
+    if (draft) {
+      id = id.substring(6);
+    }
+    if (!Object.hasOwn(webFeatures, id)) {
+      items.push(draft ? `${id} (draft)` : id);
+    }
+  }
+
+  return {
+    heading: "web-platform-tests",
+    items,
+    trailer: `Source: [WEB_FEATURES_MANIFEST.json](${asset.browser_download_url})`,
```

Comment:
This link isn't especially useful, but linking directly to the WEB_FEATURES.yml file that has the reference would require fetching and parsing all of them, which I don't want to do.